CI/stable build failure: wiremock 0.6.5 fails with E0658 (let-chain unstable)

## Summary
`cargo test` and `cargo clippy` fail on stable Rust because `wiremock v0.6.5` does not compile (`E0658: let expressions in this position are unstable`). This currently blocks local verification and CI.

## Environment
- Repo: `EYH0602/skillshub`
- rustc: `1.86.0 (05f9846f8 2025-03-31)`
- cargo: `1.86.0 (adf9b6ad1 2025-02-28)`
- CI currently uses `stable` (`.github/workflows/ci.yml`)

## Reproduction
1. Clone the repo.
2. Run:
   ```bash
   cargo clippy --all-targets --all-features -- -D warnings
   ```
   or
   ```bash
   cargo test --all-features
   ```

## Actual Behavior
Compilation fails inside `wiremock`:
```text
error[E0658]: `let` expressions in this position are unstable
   --> .../wiremock-0.6.5/src/matchers.rs:214:12
```

## Expected Behavior
Both commands should compile and run on stable Rust, matching CI configuration.

## Notes
- `Cargo.toml` currently pins `wiremock = "0.6"` under `[dev-dependencies]`.
- CI runs stable toolchain and executes these commands in `.github/workflows/ci.yml`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,6 +32,6 @@ chrono = { version = "0.4", features = ["serde"] }
 version = "3.10"
 
 [dev-dependencies]
-wiremock = "0.6"
+wiremock = "0.5"
 tokio = { version = "1", features = ["rt", "macros"] }
 serial_test = "3.0"

--- a/src/registry/github.rs
+++ b/src/registry/github.rs
@@ -1265,7 +1265,7 @@ name: minimal-skill
                 .respond_with(
                     wiremock::ResponseTemplate::new(403)
                         .insert_header("X-RateLimit-Remaining", "0")
-                        .insert_header("X-RateLimit-Reset", &reset_ts)
+                        .insert_header("X-RateLimit-Reset", reset_ts.as_str())
                         .set_body_string("rate limited"),
                 )
                 .up_to_n_times(1)

--- a/tests/common/mock_github.rs
+++ b/tests/common/mock_github.rs
@@ -118,7 +118,7 @@ impl MockGitHub {
                         "message": "API rate limit exceeded"
                     }))
                     .insert_header("X-RateLimit-Remaining", "0")
-                    .insert_header("X-RateLimit-Reset", &reset_ts),
+                    .insert_header("X-RateLimit-Reset", reset_ts.as_str()),
             )
             .mount(&self.server)
             .await;
@@ -144,7 +144,7 @@ impl MockGitHub {
                         "message": "API rate limit exceeded"
                     }))
                     .insert_header("X-RateLimit-Remaining", "0")
-                    .insert_header("X-RateLimit-Reset", &reset_ts),
+                    .insert_header("X-RateLimit-Reset", reset_ts.as_str()),
             )
             .expect(fail_count)
             .mount(&self.server)

